docs/en_US/SupportedFramework_Library.md

```diff
@@ -0,0 +1,51 @@
+# Framework and Library Supported 
+NNI provides Python API, supporting all frameworks and libraries written in python (`version >= 3.5`), and we have implemented many related examples and detailed their principles and running steps.
```

Comment:
With the built-in Python API, NNI naturally supports the hyper parameter tuning and neural network search for all the AI frameworks and libraries who support Python models. NNI had also provided a set of examples and tutorials for some of the popular scenarios to make jump start easier.

Reply:
Fixed